torchinductor/lowering.py

```diff
@@ -2173,12 +2173,14 @@ def max_pool2d_with_indices_backward(
             h // stride[0] - max(0, (h - kernel_size[0]) // stride[0])
             for h in range(kernel_size[0] * 2)
         ]
+        + [1]
```

Comment:
Why does adding [1] here fix it?

Reply:
It's to make h_window_size/w_window_size at least has value 1. I can rewrite this to use another `max` if that makes things more clear.